build.sbt

```diff
@@ -360,4 +361,17 @@ lazy val docs = project("docs")
   )
   .settings(ParadoxSupport.paradoxWithCustomDirectives)
 
+lazy val compatibilityTests = Project("akka-http-compatibility-tests", file("akka-http-compatibility-tests"))
+  .enablePlugins(NoPublish)
+  .disablePlugins(BintrayPlugin, MimaPlugin)
+  .settings(
+    libraryDependencies += "com.typesafe.akka" %% "akka-http" % "10.1.8" % "provided", // TODO, should we make that latest?
```

Comment:
Probably?

Reply:
Maybe let's keep it at 10.1.8 for a while which was a version that was alive for a long time.